src/main/java/net/dv8tion/jda/bot/sharding/ShardManager.java

```diff
@@ -918,8 +932,8 @@ default VoiceChannel getVoiceChannelById(final String id)
      * @param  id
      *         The id of the target shard
      *
-     * @throws IllegalArgumentException
-     *         if shardId is negative or higher than maxShardId
+     * @throws java.lang.IllegalArgumentException
```

Comment:
This was fine without since its `java.lang` anyway

Reply:
I thought so, but the rest of the documentation specified `java.lang` as well so I added it for the sake of consistency.